### PR TITLE
chore(DVPN-178): regex generation SQL string formatting tweaks

### PR DESCRIPTION
**Pull request by kik-kik**
# chore(DVPN-178): regex generation SQL string formatting tweaks

### CODE REVIEW

File: generator/views/funnel_analysis_view.py [hunk 1]
```diff
@@ -104,19 +104,22 @@ def _funnel_analysis_lookml(self) -> List[Dict[str, Any]]:
                 "name": f"completed_step_{n}",
                 "type": "yesno",
                 "description": f"Whether the user completed step {n} on the associated day.",
-                "sql": "".join(
+                "sql": "\n".join(
```
Comment (fbertsch) [inline generator/views/funnel_analysis_view.py:107]: Might be best to use `dedent` and a multi-line string here instead

Comment (kik-kik) [reply generator/views/funnel_analysis_view.py:107]: yes, I'll push that change tomorrow.

File: generator/views/funnel_analysis_view.py [hunk 2]
```diff
@@ -104,19 +105,22 @@ def _funnel_analysis_lookml(self) -> List[Dict[str, Any]]:
...
+REGEXP_CONTAINS(
+    ${{TABLE}}.events, mozfun.event_analysis.create_funnel_regex(
+        [{", ".join([
+            f'${{step_{ni}.match_string}}' for ni in range(1, n + 1)
+        ])}],
+        True
+    )
+)
+"""
+                )
```
Comment (fbertsch) [inline generator/views/funnel_analysis_view.py:119]: Do you agree that this indenting is easier to read? And makes use of the `dedent` function
```suggestion
                "sql": dedent(f"""
                              REGEXP_CONTAINS(
                                  ${{TABLE}}.events, mozfun.event_analysis.create_funnel_regex(
                                      [{", ".join([
                                          f'${{step_{ni}.match_string}}' for ni in range(1, n + 1)
                                      ])}],
                                      True
                                  )
                              )
                              """)
```

**Review by fbertsch — APPROVED** [review]
LGTM, but I would re-indent the SQL to keep it readable. See e.g. [here](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/tests/schema/test_schema.py#L55)

**Pull request merged**